doc/builders.rst

```diff
@@ -341,6 +341,18 @@ for details.
 
    .. autoattribute:: supported_image_types
 
+.. module:: sphinx.builders.dummy
+.. class:: DummyBuilder
+
+   This builder produces no output.  The input is only parsed and checked for
+   consistency.  This is useful for linting purposes.
+
+   .. autoattribute:: name
+
+   .. autoattribute:: supported_image_types
```

Comment:
I think so. Sometime I want to know which formats are supported by a builder.
In this case, the documentation will indicate the dummy builder supports no image format explicitly.